docs/source/entities-advanced.mdx

```diff
@@ -329,20 +488,16 @@ Follow these steps to use the `@override` directive:
 
 3. Update your router's supergraph schema to migrate to the updated Billing subgraph.
 
-    * **To migrate gradually with progressive `@override`**, you must use the Apollo Router with an Enterprise license. We recommend using progressive `@override` if it's available.
-
-       To use progressive `@override`, add a `label` argument to the `@override` directive with the percentage of requests that you want to be resolved by the subgraph. For example:
-
-        ```graphql title="Billing subgraph"
-        type Bill @key(fields: "id") {
-          id: ID!
-          amount: Int! @override(from: "Payments", label: "percent(15)") #highlight-line
-        }
-        ``` 
+   * Use progressive `@override` by adding a `label` argument to the `@override` directive with the percentage of requests that you want to be resolved by the subgraph. For example:
 
-        Setting `label: "percent(15)"` means that 15 percent of the requests for `amount` are resolved by the Billing subgraph, while the remaining 85 percent are resolved by the Payments subgraph. 
+      ```graphql title="Billing subgraph"
+      type Bill @key(fields: "id") {
+        id: ID!
+        amount: Int! @override(from: "Payments", label: "percent(15)") #highlight-line
+      }
```

Comment:
The `label` arg should be explained and used initially, so up in step 2 when we first add the override directive. Just for the sake of example, let's use `percent(1)` to illustrate how we should start out small with the migration.